README.md

```diff
@@ -50,3 +50,13 @@ jenkins-jobs --conf /path/to/your/jenkins_jobs.ini update /path/to/your/template
 ```
 ./scripts/configure_jenkins /path/to/your/jenkins_jobs.ini
 ```
+
+## Troubleshooting
+To prevent from getting SSL: CERTIFICATE_VERIFY_FAILED when running jenkins-jobs commands:
+`ssl.SSLCertVerificationError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: self signed certificate in certificate chain`
+
+execute the following command in the terminal:
+
+```
+export PYTHONHTTPSVERIFY=0
```

Comment:
This is really a temporary solution. Have you tried installing Red Hat IT CA certificate?

Reply:
installing certificate did not work for me

Reply:
@pawelpaszki @matskiv I would vote for merging. I can't reproduce on my machine. I can't remember if I did some special setup/config of python to make it work. I don't have `PYTHONHTTPSVERIFY` defined.

Reply:
I would probably merge it as well. It might not be the best solution, but it works for me